lib/linux_admin/hosts.rb

```diff
@@ -36,6 +38,10 @@ def update_entry(address, hostname, comment = nil)
       end
     end
 
+    def hostname=(name)
+      run!(cmd('hostnamectl'), :params => ['set-hostname', name])
```

Comment:
Yeah, it's `--file` so I'll use `:params => {:file => "/etc/hostname"}`

But on a side note:
```ruby
irb(main):004:0> AwesomeSpawn::CommandLineBuilder.new.build("hostname", {:F => "/etc/hostname"})
=> "hostname --F /etc/hostname"
```
Am I missing something?